awx/api/serializers.py

```diff
@@ -4799,6 +4799,7 @@ class Meta:
             "mem_capacity",
             "enabled",
             "managed_by_policy",
+            "node_type",
```

Comment:
Should be read-only.

Reply:
If so, how can we convert a hybrid node that was created in 4.0 into a control-only node?

Reply:
what if we set editable=False on the field, and then `awx-manage provision_instance` could still change it to reflect value in the inventory file?

Reply:
We made the provision_instance command be able to change the type.  Resolved.